internal/db_methods/db_methods.go

```diff
@@ -11,11 +11,29 @@ import (
 )
 
 var (
-	payloadsFields        = []string{"payloads.id", "payloads.request_id", "payloads.account", "payloads.request_id", "payloads.system_id"}
+	payloadsFields        = []string{"payloads.id", "payloads.request_id", "payloads.account", "payloads.request_id", "payloads.system_id", "payloads.inventory_id"}
 	payloadStatusesFields = []string{"payload_statuses.status_msg", "payload_statuses.date", "payload_statuses.created_at"}
 	otherFields           = []string{"services.name as service", "sources.name as source", "statuses.name as status"}
 )
 
+func defineVerbosity(verbosity string) string {
+	switch verbosity {
+	case "0":
```

Comment:
I think if we set a variable for verbosity to default to 0, we won't need this stanza here. I'll comment where I think it should go